### DISCUSSION TITLE
smartthings-govee-led-lights - update regarding API polling

### DISCUSSION THREAD

**Discussion by uncdubdiver** [uncdubdiver/smartthings-govee-led-lights#16] [General]
Just a heads up to everyone, when Govee updated their API to limit the number of requests allowed to their API, this drastically affected the integration with their API.

I've implemented another field/parameter that you can set to change the default polling amount of time (default is every 2min, which if you have multiple Govee LED strips, you might hit this limit faster than you expect).

Look for this to come out soon after further testing.

**Comment by nothinghr** [comment]
my integration just stopped working a couple of days ago, is Smartthings preventing now the old handlers to work?

**Comment by uncdubdiver** [comment]
Unfortunately I do not know the answer to this. From what I heard they stopped supporting it’s drivers as of Dec 31 2022, however if it still worked then that’s great, but I doubt it’ll work for much longer.

 

With that being said, you can also check the value you have set for the Polling/Refresh Time field to make sure it’s not very low. The lower it is, the more times it’ll hit the Govee API and Govee has been cracking down and limiting the # of requests allowed/day, so this could be causing your problem. I’ve got mine set to “600” seconds and even that causes some issues sometimes ☹

 

Thank You.

 

Dean Berman
DB Development
***@***.*** ***@***.***> 
(O) 704.837.0284
(C) 704.577.1236
